RSEConUK2019-Cylc-Talk/js/running-workflows-mo-research-plot.js

```diff
@@ -0,0 +1,62 @@
+// Data in 'data/running_workflows_data.js', imported via HTML before this runs
+
+// Convert raw data from workflow analysis into form required by Plotly API.
+var FOMARTTED_RUNNING_WORKFLOWS_DATA = {
+  x: [],
+  y: [],
+  mode: 'Scatter',
+  line: {
+    color: '#00C798',
+    width: 2
+  }
+};
+for (dataPoint of rawRunningWorkflowData.data) {
+  FOMARTTED_RUNNING_WORKFLOWS_DATA.x.push(dataPoint[0]);
+  FOMARTTED_RUNNING_WORKFLOWS_DATA.y.push(dataPoint[1]);
+}
+
+RUNNING_WORKFLOWS_PLOTTED =
```

Comment:
Need to declare constants/variables:

```suggestion
const RUNNING_WORKFLOWS_PLOTTED =
```

(I flag this because I've hit issues with undeclared JS variables behaving differently in different browsers.)

Reply:
Good spot, no need to justify raising it, it is probably important (I was aware JS needs declaration but forgot in this instance, I still can't fully shake myself out of Python mode!)